.travis.yml

```diff
@@ -0,0 +1,14 @@
+branches:
+  except:
+    - master
+language: ruby
+cache: bundler
+sudo: false
+rvm:
```

Comment:
Exactly what we need indeed. I can cut out kitchen gems too since we don't use those in travis.

I had no success installing guard on 2.1.8 or 2.1.10. They say bundler should be able to resolve it, but it doesn't. So specifying the args is necessary unless I remove the guard gems from the bundle entirely (seems weird to me given the Guardfile referencing Guard gems for RSpec and Foodcritic)